packages/driver/package.json

```diff
@@ -16,6 +16,9 @@
   "files": [
     "lib"
   ],
+  "dependencies": {
+    "diff": "^4.0.1"
```

Comment:
I would use exact dependency to stop accidental upgrades when we build Cypress and can rebuild the same binary (mostly)

Reply:
agreed, I'll change this now!